yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/AttributeConverter.java

```diff
@@ -47,32 +50,59 @@ static final AttributeConverter newInstance() {
     }
 
     private Object convertValueFromProto(AttrProto.Attribute attribute) throws ParseException, IOException {
-        switch (attribute.getContentType()) {
+        return convertValue(attribute.getContentType(), attribute.getValue());
```

Comment:
This method doesn't really add anything does it? Now that we're just calling the method below

Reply:
I think this is so that the switch logic in `convertValue()` can be reused by `convertMultiValue()` without breaking existing calls to `convertValueFromProto()`, which expects `AttrProto.Attribute`.

`convertMultiValue()` needs to pass `ContentTypeProto.ContentType` to `convertValue()` (not `AttrProto.Attribute`)

Reply:
Not much, no.

I just thought maybe leaving it might make it clearer that there are two different ways in to _convertValue_.  Maybe just more obvious / more readable.  This class is starting to get larger, there's more to follow.

Can remove it if you want.

Reply:
Is the alternative to change the arguments of `convertValueFromProto()` and change line 44:
```java
Object value = convertValueFromProto(attribute);
```
to:
```java
Object value = convertValueFromProto(attribute.getContentType(), attribute.getValue());
```
Instead?

Reply:
> Is the alternative to change the arguments of `convertValueFromProto()` and change line 44:
> 
> ```java
> Object value = convertValueFromProto(attribute);
> ```
> 
> to:
> 
> ```java
> Object value = convertValueFromProto(attribute.getContentType(), attribute.getValue());
> ```
> 
> Instead?

That's what I was thinking. We don't need the whole attribute object since we're only using `contentType` and `value`. I think it would be cleaner if we removed this method, but it's up to you @bucky-boy!